client/src/featureform/resources.py

```diff
@@ -1397,5 +1397,27 @@ def config(self):
             "Credentials": self.credentials.config(),
         }
 
+@typechecked
+@dataclass
+class SparkExecutor:
+    def __init__(self,
```

Comment:
Are there any other args we would want to allow?

Reply:
the only thing that I can think of is configurations like memory, executors, etc.